src/data.js

```diff
@@ -0,0 +1,327 @@
+    {
+      "id": 737799,
+      "poster_path": "https://image.tmdb.org/t/p/original//tEvGSlRO0dz1pINzjSEELdTHvwk.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//ysGikUSBjG3NRSyycHPf4rsIXQm.jpg",
+      "title": "Maquis",
+      "average_rating": 4.8,
+      "release_date": "2020-08-28"
+    },
+    {
+      "id": 500840,
+      "poster_path": "https://image.tmdb.org/t/p/original//5ynWWapdl45hJXUh0KIevxSG9JQ.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//x3NqAzuTWvnron4pXXyFGkyTFo7.jpg",
+      "title": "I'm Thinking of Ending Things",
+      "average_rating": 6,
+      "release_date": "2020-08-28"
+    },
+    {
+      "id": 547017,
+      "poster_path": "https://image.tmdb.org/t/p/original//iSwTnNS7TKAS79Sz9LvyqlBxxrU.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//97UR8xPpUNqpvx5zr7eyf4YSBCE.jpg",
+      "title": "Shirley",
+      "average_rating": 5.25,
+      "release_date": "2020-09-11"
+    },
+    {
+      "id": 449924,
+      "poster_path": "https://image.tmdb.org/t/p/original//swmjF0CQc59K3jdUFzOirXESeGN.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//hb3kjSQI0lwGUCIxnfAYec08VSq.jpg",
+      "title": "Ip Man 4: The Finale",
+      "average_rating": 6,
+      "release_date": "2019-12-19"
+    },
+    {
+      "id": 425001,
+      "poster_path": "https://image.tmdb.org/t/p/original//yUFbPtWeDbVR3zmqshOaL5lScyo.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//4gKyQ1McHa8ZKDsYoyKQSevF01J.jpg",
+      "title": "The War with Grandpa",
+      "average_rating": 5.666666666666667,
+      "release_date": "2020-01-31"
+    },
+    {
+      "id": 582885,
+      "poster_path": "https://image.tmdb.org/t/p/original//ow3OA6OC0rjgbMVaLn2drwjvqZE.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//6j2g4fEAmjJqAclzprIpvz9Gaw4.jpg",
+      "title": "Cuties",
+      "average_rating": 3.5,
+      "release_date": "2020-08-19"
+    },
+    {
+      "id": 659991,
+      "poster_path": "https://image.tmdb.org/t/p/original//erl30EcM8b8S84mvw8QXhNIeSfi.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//5nRyaVklxyA9OkxqZaPv1KBRqpd.jpg",
+      "title": "Made in Italy",
+      "average_rating": 5,
+      "release_date": "2020-08-06"
+    },
+    {
+      "id": 501953,
+      "poster_path": "https://image.tmdb.org/t/p/original//2wXrBtfrvwMWE1i3iHjKjoRZjYk.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//oeaLQKoPFQxvhEz3yyR1QuestXG.jpg",
+      "title": "Eternal Beauty",
+      "average_rating": 3,
+      "release_date": "2020-08-21"
+    },
+    {
+      "id": 585244,
+      "poster_path": "https://image.tmdb.org/t/p/original//dqA2FCzz4OMmXLitKopzf476RVB.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//21Q8bzu10YF9i4O5amBkJBombYo.jpg",
+      "title": "I Still Believe",
+      "average_rating": 3.8333333333333335,
+      "release_date": "2020-03-12"
+    }
+  ]
+};
+
+export default movieData;
```

Comment:
I wonder how this dummy data file will come into play with Cypress later, if at all 🤔